suzieq/poller/worker/services/service.py

```diff
@@ -813,6 +813,11 @@ async def run(self):
             # Post a cmd to fire up the next poll after the specified period
             self.logger.debug(
                 f"Rescheduling service for {self.name} service")
+            if self.run_once == "update":
```

Comment:
Just a detail: logging `Rescheduling service` when the mode is `update` is not correct

Reply:
Ack, good catch. FIxing it.